modules/core/src/main/java/org/apache/synapse/rest/RESTConstants.java

```diff
@@ -85,4 +85,10 @@
                                                                     _SYNAPSE_INTERNAL_+ "Access-Control-Allow-Headers";
     public static final String INTERNAL_CORS_HEADER_ORIGIN = _SYNAPSE_INTERNAL_+ "Origin";
 
+    /**
+     * Constant related to identifying API when Prometheus Observability Handler is present
+     */
+    public static final String IS_ALREADY_PROCESSED = "IS_ALREADY_PROCESSED";
```

Comment:
How about IS_PROMETHEUS_ENGAGED? Just a suggestion because 'IS_ALREADY_PROCESSED' seems a bit too generic.

Reply:
Shouldn't this constant match with what is in Micro-Integrator https://github.com/wso2/micro-integrator/blob/8d6762305b7f61716fcb6c929e7387467d454587/components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.observability/src/main/java/org/wso2/micro/integrator/prometheus/observability/metrics/util/PrometheusMetricsConstants.java#L44

Reply:
Fixed